### PR TITLE
Remove cruft from upload results page

### PR DESCRIPTION
**Pull request by zarino**
Includes @crowbot’s one-line fix to the missing JavaScript block, from #23.

The tabular data preview has been moved nearer the top of the "Upload results" page, and all the rest of the "below the fold" content from `explore_base.html` and `explore_release.html` has been removed.

Screenshot_2020-09-02 Screenshot

Screenshot_2020-09-02 Screenshot(1)

@SimKennedy there’s still a _huge_ chunk of JavaScript at the bottom of `explore_base.html`. I haven’t removed it because I’m not sure whether it’s used anywhere? I can tell it’s _not_ used in the `/data/…` pages I‘ve been testing, but I can’t work out where else `explore_base.html` might be used.

### CODE REVIEW

File: cove_ocds/templates/cove_ocds/explore_release.html [hunk 1]
```diff
@@ -104,6 +104,111 @@ <h1>Upload results</h1>
...
+                        <li>
+                            <strong>{% blocktrans %}Procuring Entity:{% endblocktrans %}</strong>
+                          {% if release.tender.procuringEntity.name %}
+                            {{ release.tender.procuringEntity.name }}
+                          {% endif %}
+                          {% if release.tender.procuringEntity.identifier.id %}
+                            ({{ release.tender.procuringEntity.identifier.id }})
+                          {% endif %}
+                        </li>
+                      {% endif %}
```
Comment (SimKennedy) [inline cove_ocds/templates/cove_ocds/explore_release.html:204]: We won't be mapping to procuringEntity so just the Buyer info and header is good.
```suggestion
```

**Review by SimKennedy — CHANGES_REQUESTED** [review]
Looking good @zarino. A few things to mention:
The feedback should be in context of the simple CSV data submitted, and avoiding OCDS terminology where possible.

For reference, some sample files are here: `silvereye/data/cf_daily_csv/sample_submissions`
And the mappings are here: `silvereye/data/csv_mappings/release_mappings.csv`

So I think we can change some fields 
- OCID: we generate this from the given `release/id` so let's show that instead
- Tags: all rows should be a single type within a file which we should show in the summary at the top, so not needed in the table.
- Buyer: The buyer (for now) _should_ be expected to be the same for all rows in a submission and the same as the publisher which we dispaly above, so we could hide this for now. (In theory this may change in the future and a publisher can publish for multiple buyers, so keep the code?)

As for the javascript, that `explore_base.html` is only used in `explore_record.html` which we are not using in Silvereye, so if it's not needed lets clear it.

But, it raises a point about organisation, perhaps we could move our modified templates in to `silvereye/templates/silvereye/explore_*` and put the originals back for reference. Then just update the view?
https://github.com/spendnetwork/cove-ocds/blob/f28e899a6cbdcd526ed44235415bd2e4023b4c63/cove_ocds/views.py#L379

**Comment by zarino** [comment]
@SimKennedy I’ve made the changes we discussed. Let me know what you think.

**Review by SimKennedy — APPROVED** [review]

**Comment by zarino** [comment]
@SimKennedy this looks good to me! Well done sorting out the Bootstrap 4 module :+1:

**Pull request merged**